extensions/resteasy-reactive/quarkus-jaxrs-client/deployment/src/main/java/io/quarkus/resteasy/reactive/client/deployment/JaxrsClientProcessor.java

```diff
@@ -222,6 +226,9 @@ public void registerInvocationCallbacks(CombinedIndexBuildItem index, ResteasyRe
                                             String.class, Object[].class),
                                     tg, m.load(p.name), array);
                         }
+                        if (p.parameterType == ParameterType.BODY) {
```

Comment:
This should really be an 'else if', the end result is the same but it make the code easier to read.